.packit.yaml

```diff
@@ -5,15 +5,19 @@ synced_files:
 upstream_package_name: fmf
 downstream_package_name: fmf
 
+# Epel9 fails to build with dynamic version. Need to create archive with PKG-INFO
+# F37 works with setuptools_scm 7.0
 actions:
   create-archive:
-  - make tarball
+    - "python3 -m build --sdist --outdir ."
```

Comment:
Indeed, I agree. I only made the comment because the hatch is already there to get version and perhaps that the epel9 packaged `build` could not be as up-to-date as I know hatch is and could be causing issues like missing PKG-INFO.